### PR TITLE
Use util.promisify instead of Bluebird.promisify

### PR DESCRIPTION
**Pull request by maximelkin**
fixes https://github.com/tgriesser/knex/issues/3257

### CODE REVIEW

**Comment by maximelkin** [comment]
Not sure what to do with Bluebird.promisifyAll in https://github.com/tgriesser/knex/blob/master/lib/dialects/oracle/index.js#L85

File: lib/migrate/MigrationGenerator.js [hunk 1]
```diff
@@ -30,9 +30,7 @@ class MigrationGenerator {
     const dirs = this._absoluteConfigDirs();
 
     const promises = dirs.map((dir) => {
-      return bluebird
-        .promisify(fs.stat, { context: fs })(dir)
-        .catch(() => bluebird.promisify(mkdirp)(dir));
+      return promisify(fs.stat)(dir).catch(() => promisify(mkdirp)(dir));
```
Comment (kibertoad) [inline lib/migrate/MigrationGenerator.js]: `{ context: fs }` part is irrelevant?

Comment (maximelkin) [reply lib/migrate/MigrationGenerator.js]: yes, but i cant find it in official docs
https://nodejs.org/docs/latest-v8.x/api/util.html#util_util_promisify_original

**Comment by kibertoad** [comment]
Bunch of conflicts came up.

**Comment by kibertoad** [comment]
Thanks!

**Pull request merged**